tests/Functional/SQL/ParserTest.php

```diff
@@ -17,4 +17,24 @@ public function testPostgreSQLJSONBQuestionOperator(): void
 
         self::assertTrue($this->connection->fetchOne('SELECT \'{"a":null}\'::jsonb ?? :key', ['key' => 'a']));
     }
+
+    /** test the REPLACE_PATTERNS change for ARRAY [ to ARRAY[ does not cause the mixed positional and named parameters error found prior to the fix */
+    public function testParametersInArrayConstructWithWhitespace(): void
+    {
+        if (! TestUtil::isDriverOneOf('pdo_pgsql')) {
```

Comment:
Please include the `pgsql` driver as well.

Reply:
👍